resources/js/components/shared/IconDropdown.vue

```diff
@@ -160,9 +160,14 @@ $multiselect-height: 33px;
   }
 
   .multiselect__single {
-    font-size: 15px;
+    font-size: 16px;
```

Comment:
Please apply this same size in the Select chart

Reply:
![image](https://github.com/ProcessMaker/processmaker/assets/42388723/d62913ca-c027-41bc-896e-3f8e17f2cc18)

Reply:
Required changes were applied:
![image](https://github.com/ProcessMaker/processmaker/assets/131476081/4ee6b8c5-2fcd-471e-89a1-d46c30348e58)